.codeclimate.yml

```diff
@@ -27,7 +27,7 @@ checks:
 plugins:
   rubocop:
     enabled: true
-    channel: rubocop-1-9
+    channel: rubocop-1-9-1
```

Comment:
![Screenshot 2021-02-10 at 14 56 53](https://user-images.githubusercontent.com/206108/107519233-4de1ec80-6bb0-11eb-8da0-c25fc7caf53a.png)

Reply:
![Screenshot 2021-02-10 at 15 11 39](https://user-images.githubusercontent.com/206108/107520985-4de2ec00-6bb2-11eb-852c-3b000dee0734.png)
![Screenshot 2021-02-10 at 15 12 07](https://user-images.githubusercontent.com/206108/107521041-5c310800-6bb2-11eb-8a86-39663ff4a067.png)


ugh, this "fix" breaks codeclimate - I'll better outsource this into a separate PR :)

Reply:
👉 #8991